### PR TITLE
Aligning hertz configuration items with gofiber

### PR DESCRIPTION
**Pull request by Skyenought**
### Description

In `hertz` 0.7.2, the `disableDate` and `disableContentType` configuration options have been added.

This commit aims to bring hertz configuration closer to gofiber.

### CODE REVIEW

File: go/hertz/main.go [hunk 1]
```diff
@@ -12,6 +12,8 @@ func main() {
 		server.WithHostPorts(":3000"),
 		server.WithDisableHeaderNamesNormalizing(true),
 		server.WithDisablePrintRoute(true),
+		server.WithDisableDefaultDate(true),
+		server.WithDisableDefaultContentType(true),
```
Comment (waghanza) [inline go/hertz/main.go:16]: Not sure this one should be here (maybe it should be removed from fiber too)

**Review by waghanza — CHANGES_REQUESTED** [review]
Thanks for your contribution :tada:

File: go/hertz/main.go (hunk 1 above)
Comment (Skyenought) [reply go/hertz/main.go:16]: GET. 
If it is determined that alignment is not needed, I turn off PR.

Comment (waghanza) [reply go/hertz/main.go:16]: Feel free to remove this kind of option in all frameworks (including `fiber`

**Review by waghanza — DISMISSED** [review]

**Comment by waghanza** [comment]
weird commpilation fail on `fiberè

**Review by waghanza — APPROVED** [review]

**Pull request merged**